decoration.sk

```diff
@@ -16,6 +16,76 @@ unchanged decoratives:
 	item frame [item]¦s = minecraft:item_frame[relatedEntity=item frame]
 	armor stand [item]¦s = minecraft:armor_stand[relatedEntity=armor stand]
 
+village and pillage decoratives:
+	minecraft version = 1.14 or newer
+	{waterloggable} {directional} {wood type} wall sign¦s = -wall_sign
+	{waterloggable} {rotatable} {wood type} floor sign¦s = -sign
+	[any] sign¦s = wall sign, floor sign
+
+	cornflower¦s = minecraft:cornflower
+	lily of the valley = minecraft:lily_of_the_valley
+	wither rose¦s = minecraft:wither_rose
+
+	{waterloggable} brick wall¦s = minecraft:brick_wall
+	{waterloggable} prismarine wall¦s = minecraft:prismarine_wall
+	{waterloggable} sandstone wall¦s = minecraft:sandstone wall
+	{waterloggable} red sandstone wall¦s = minecraft:red_sandstone_wall
+	{waterloggable} stone brick wall¦s = minecraft:stone_brick_wall
+	{waterloggable} mossy stone brick wall¦s = minecraft:mossy_stone_brick_wall
+	{waterloggable} granite wall¦s = minecraft:granite_wall
+	{waterloggable} diorite wall¦s = minecraft:diorite_wall
+	{waterloggable} andesite wall¦s = minecraft:andesite_wall
+	{waterloggable} nether brick wall¦s = minecraft:nether_brick_wall
+	{waterloggable} red nether brick wall¦s = minecraft:red_nether_brick_wall
+	{waterloggable} end stone brick wall¦s = minecraft:end_stone_brick_wall
+
+	{scaffold bottom}:
+		{default} = -
+		bottomless = -[bottom=false]
+		bottom = -[bottom=true]
+	{waterloggable} {scaffold bottom} scaffolding = minecraft:scaffolding
+
+	#Crafting Blocks
+	{directional} loom¦s = minecraft:loom
+	{barrel open}:
+		{default} = -
+		closed = -[open=false]
+		open(ed) =[open=true]
+	{orientable} {barrel open} barrel¦s = minecraft:barrel
+	{directional} {lightable} smoker¦s = minecraft:smoker
+	{directional} {lightable} blast furnace¦s = minecraft:blast_furnace
+	{directional} stone[ ]cutter¦s = minecraft:stonecutter
+
+	#Whether the grindstone is on the floor, wall, or ceiling.
+	{grindstone face}:
+		{default} = -
+		floor = -[face=floor]
+		wall = -[face=wall]
+		ceiling - [face=ceiling]
+	{directional} {grindstone face} grindstone¦s = minecraft:grindstone
+
+	#Non-functional "crafting" blocks
+	cartography table¦s = minecraft:cartography_table
```

Comment:
Not that this makes any difference, but the cartography table **does** have a function

Reply:
Oops! Must have accidentally moved that one when I was sorting through them.